test/parallel/test-crypto-key-objects.js

```diff
@@ -168,6 +168,9 @@ const privatePem = fixtures.readSync('test_rsa_privkey.pem', 'ascii');
   }, /null/);
 }
 
+if (!/1\.1\.1/.test(process.versions.openssl))
```

Comment:
I think this check will need to be more elaborate for versions like `2.0.0` or `1.2.0` or `1.1.2`. Perhaps something like:

```js
const eddsaSupported = (() => {
  const sslVer = /^(\d+)\.(\d+)\.(\d+)/.exec(process.versions.openssl);
  if (sslVer) {
    const major = +sslVer[1];
    if (major > 1)
      return true;
    if (major === 1) {
      const minor = +sslVer[2];
      if (minor > 1)
        return true;
      if (minor === 1) {
        const patch = +sslVer[3];
        return (patch >= 1);
      }
    }
  }

  return false;
})();
```

Reply:
Same for the other test that does a similar check.

Reply:
If this line was on master, I'd agree, but remember this line of code exists only for 11.x (and 10.x eventuall), and they are not going to get updated past openssl 1.1.1. 10.x isn't getting updated to 3.0.0 (when it comes), and 11.x is going out of maintenance in a few months, before 3.0.0 will exist. So, there are only two options, 1.1.0, or 1.1.1.

Reply:
used a different check, and most of this  was for edwards curves, which are now gone from v11.x-staging